frontend/app/styles/partials/signup.scss

```diff
@@ -48,6 +48,11 @@ $colorwhite: #ffffff;
   }
 }
 
+small {
+    color: red;
```

Comment:
there are some indentation errors as well that may cause problems in future and so travis is failing as well. 
please fix them and ask for a review again.

Reply:
will fix it asap.